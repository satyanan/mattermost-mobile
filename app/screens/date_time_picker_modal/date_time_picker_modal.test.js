// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.
import React from 'react';

import {shallowWithIntl} from 'test/intl-test-helper';

import DateTimePickerModalBase from './date_time_picker_modal_base';

describe('DateTimePickerModalBase', () => {
    const navigator = {
        setOnNavigatorEvent: jest.fn(),
        setButtons: jest.fn(),
    };
    const baseProps = {
        deviceHeight: 320,
        deviceWidth: 120,
        navigator,
        onChange: jest.fn(),
        minimumDate: new Date(),
    };

    test('should match snapshot', () => {
        const wrapper = shallowWithIntl(
            <DateTimePickerModalBase {...baseProps}/>,
        );

        expect(wrapper.getElement()).toMatchSnapshot();
    });
});
