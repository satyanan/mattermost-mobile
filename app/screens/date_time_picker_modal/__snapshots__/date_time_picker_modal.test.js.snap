// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateTimePickerModalBase should match snapshot 1`] = `
<TouchableWithoutFeedback
  onPress={[Function]}
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(0, 0, 0, 0.8)",
        "bottom": 0,
        "flex": 1,
        "position": "absolute",
      }
    }
  >
    <AnimatedComponent
      style={
        Object {
          "height": 320,
          "left": 0,
          "top": 320,
          "width": 120,
        }
      }
    >
      <DatePickerIOS
        date={2019-10-09T11:29:01.253Z}
        initialDate={2019-10-09T11:29:01.253Z}
        is24Hour="true"
        minimumDate={2019-10-09T11:29:01.404Z}
        minuteInterval={30}
        mode="datetime"
        onDateChange={[Function]}
      />
      <Button
        onPress={[Function]}
        title="Ok"
      />
    </AnimatedComponent>
  </View>
</TouchableWithoutFeedback>
`;
